Create Bishop moves  for Dark pieces

Mrsatatima opened the pull request:
made changes and updated for your review thank you

Mrsatatima committed:
Create Bishop moves  for light pieces

A function for Bishop moves, it generates all possible for Bishop pieces
while making sure pieces doesn't go off board and jump over pieces
it captures enemy pieces (dark) and stop at that square.

drak00 committed:
FIX ZeroDivisionError in animate

Mrsatatima committed:
Merge branch 'light_team' of github.com:drak00/Chess into light_team

Mrsatatima committed:
Create Bishop moves  for Dark pieces

A function for Bishop moves, it generates all possible for Bishop pieces
while making sure pieces doesn't go off board and jump over pieces
it captures enemy pieces (light) and stop at that square.

Bilaerl commented on engine.py:
```diff
@@ -89,23 +89,38 @@ def get_bishop_moves(self, r, c, moves):
 		##TODO
 		if self.light_to_move:
-			direction = [(-1,-1), (1,1),(1,-1), (-1,1)] # possible direction
+			direction = ((-1,-1), (+1,+1), (+1,-1), (-1,+1)) # possible direction
```
move line 92 outside _if self.light_to_move_, so that it will work for both light and dark pieces and you don't have to repeat yourself on line 108

Bilaerl reviewed (changes requested):
Nice work
What I don't understand though is why you changed all 1s to +1s in your tuple

**A suggestion:** use gitignore to stop git from tracking your pycache folder

Mrsatatima committed:
Remove __pycache__/engine.cpython-38.pyc from Repo

Mrsatatima committed:
Update Bishop move func to remove Unwanted line

Direction a list of tuple change to tuple of tuples
and move to above first if statement to avoid repetition of code
